### ISSUE TITLE
Unlabeled buttons on screen reader

### ISSUE THREAD

**Issue by azurejoga** [flarum/core#4077]
### Current Behavior

Hi, some unlabeled buttons like start a discussion button and publish post/discussion button after the title and after the recording field, these buttons are not labeled. What file can I edit to fix these issues until an official fix comes out? I'm using NVDA screen reader on the computer with windows,  but the behavior is the same for google talkback on android. I'm using the latest version of Flarum

### Steps to Reproduce

1. activate the screen reader.
2. open the google chrome
3. go to the main page before the categories/tags button and start a discussion, this button is also unlabeled
4. click and the publish button is also without a label

### Expected Behavior
The screen reader can perfectly read the buttons on Google Chrome

### Environment

- Flarum version: 1.8.7PHP
- Website URL: https://forum.eternal-legend.com.br
- Webserver: 8.1.2-1ubuntu2.19MySQL
- Hosting environment: [ vps]
- PHP version: 8.2.24
- Browser: [Chrome latest version]

### Output of `php flarum info`

```
Output of "php flarum info", run this in terminal in your Flarum directory.
```

### Possible Solution

Add the label element to the button

### Additional Context

en microsoft edge, works perfectly and is read successfully... so the bug for now is only in the google chrome, edge does not present accessibility problems

**Comment by oshaposhnyk** [comment]
Can I take this issue? This is my first issue, I want to start contributing to flarum. Thank you.

**Comment by luceos** [comment]
> Can I take this issue? This is my first issue, I want to start contributing to flarum. Thank you.

Yes you can. If you're comfortable taking it on, feel free to work on it. For new contributors it's usually smart to create a so-called Draft PR as soon as they can. Make sure to fill out the details asked for while creating the Pull Request as well, in specific linking to this issue.

Hope to see your contribution soon and thanks for offering your help, it's much appreciated 🙏

**Comment by azurejoga** [comment]
I would be very happy@oshaposhnyk

**Comment by claudiushenrichs** [comment]
New discussion button received an aria-label meanwhile: https://github.com/flarum/framework/pull/4758
